internal/method/reference_processor.go

```diff
@@ -115,17 +118,25 @@ func (rp *ReferenceProcessor) Process(_ *types.Named, f *types.Var, _, comment s
 	refType := refTypeValues[0]
 	isPointer := false
 	isList := false
+	isFloatPointer := false
+
 	// We don't support *[]string.
 	switch t := f.Type().(type) {
-	// *string
+	// *string|float64
 	case *types.Pointer:
 		isPointer = true
-	// []string.
+		if f.Type().String() == "*float64" {
+			isFloatPointer = true
+		}
+	// []string|float64.
 	case *types.Slice:
 		isList = true
-		// []*string
+		// []*string|float64
 		if _, ok := t.Elem().(*types.Pointer); ok {
 			isPointer = true
+			if f.Type().String() == "[]*float64" {
```

Comment:
nit: could we use variable `t` here 🤔

Reply:
I had an issue with the conditional around `[]*float64` when it wasn't in this string format which is why I did this approach, if you think it is worth revisiting that I can spend some time trying to get `t` working

Reply:
> if you think it is worth revisiting that I can spend some time trying to get t working

no, this was more of a nitpick, especially since this is executed during code generation not at runtime.